misc/telemetry/docker-compose.yml

```diff
@@ -21,34 +22,85 @@ services:
       - ./prometheus/prometheus.yml:/etc/prometheus/prometheus.yml
     networks:
       - gnoland-net
+  
   grafana:
-    image: grafana/grafana-enterprise
+    image: grafana/grafana
+    environment:
+      - GF_AUTH_ANONYMOUS_ENABLED=true
+      - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
     volumes:
       - grafana_data:/var/lib/grafana
-      - ./grafana/datasources.yaml:/etc/grafana/provisioning/datasources/datasources.yaml
-      - ./grafana/dashboards.yaml:/etc/grafana/provisioning/dashboards/dashboards.yaml
-      - ./grafana/gno-dashboards.json:/var/lib/grafana/dashboards/gno-dashboards.json
+      - ./grafana/provisioning:/etc/grafana/provisioning
     ports:
       - "3000:3000"
     networks:
       - gnoland-net
-  gnoland:
-    build:
-      context: ./gnoland
-      dockerfile: Dockerfile
-    ports:
-      - "26657:26657"
+  
+  gnoland-val:
+    image: ghcr.io/gnolang/gno/gnoland:master
     networks:
       - gnoland-net
+    volumes:
+      # Shared Volume
+      - gnoland-shared:/gnoroot/shared-data
+    entrypoint:
+      - sh
+      - -c
+      - |
+        gnoland secrets init
+        gnoland genesis generate
+        gnoland genesis validator add -name val000 -address $(gnoland secrets get validator_key.address -raw) -pub-key $(gnoland secrets get validator_key.pub_key -raw)
+        gnoland genesis balances add -balance-sheet /gnoroot/gno.land/genesis/genesis_balances.txt
+        gnoland genesis txs add packages /gnoroot/examples/gno.land
```

Comment:
I was about to merge it, but I guess this needs `gnogenesis` now.

Reply:
that is what we deserve now for nuking `gno genesis` sub cmd